README.md

```diff
@@ -1,16 +1,271 @@
+- **MAUI Workloads**: Install using `dotnet workload install maui`
+
+### Platform-Specific Requirements
+
+#### Windows Development
+- Windows 10 version 1903 (build 18362) or later
+- Windows App SDK for Windows target framework
+
+#### macOS Development  
+- macOS 12.0 (Monterey) or later
+- Xcode 14.0 or later for iOS/macOS targets
+
+#### Mobile Development
+- Android API level 21 (Android 5.0) or later
+- iOS 12.2 or later
+
+## Development Environment Setup
+
+### 1. Install .NET and MAUI Workloads
+
+```bash
+# Install .NET 9.0 SDK or later
+# Download from: https://dotnet.microsoft.com/download
+
+# Install MAUI workloads
+dotnet workload install maui
+
+# Verify installation
+dotnet workload list
+```
+
+### 2. Clone and Restore Dependencies
+
+```bash
+# Clone the repository
+git clone https://github.com/davidortinau/SentenceStudio.git
+cd SentenceStudio
+
+# Navigate to source directory
+cd src
+
+# Restore NuGet packages
+dotnet restore
+```
+
+### 3. API Keys Configuration
+
+Sentence Studio requires API keys for AI and text-to-speech functionality. You'll need to obtain and configure the following:
+
+#### Required API Keys
+
+1. **OpenAI API Key** - For AI-powered language learning features
+   - Visit: https://platform.openai.com/api-keys
+   - Create an account and generate an API key
+   - Ensure your account has sufficient credits or an active subscription
+
+2. **ElevenLabs API Key** - For text-to-speech functionality
+   - Visit: https://elevenlabs.io/app/speech-synthesis
+   - Create an account and generate an API key
+   - Free tier available with usage limitations
+
+#### Desktop Configuration (Development)
+
+For desktop development, set environment variables:
+
+**Windows (PowerShell):**
+```powershell
+$env:AI__OpenAI__ApiKey = "your-openai-api-key-here"
+$env:ElevenLabsKey = "your-elevenlabs-api-key-here"
+```
+
+**macOS/Linux (Terminal):**
+```bash
+export AI__OpenAI__ApiKey="your-openai-api-key-here"
+export ElevenLabsKey="your-elevenlabs-api-key-here"
+```
+
+#### Mobile Configuration
+
+For mobile deployments, create an `appsettings.json` file in the `src/SentenceStudio` project directory:
+
+```json
+{
+  "Settings": {
+    "OpenAIKey": "your-openai-api-key-here",
+    "ElevenLabsKey": "your-elevenlabs-api-key-here"
+  }
+}
+```
+
+**Important Security Note**: Never commit API keys to version control. The project's `.gitignore` excludes `appsettings.json` to prevent accidental exposure of sensitive credentials.
+
+## Building and Running
+
+### Build Commands
+
+```bash
+# Navigate to source directory
+cd src
+
+# Build for specific platforms
+dotnet build SentenceStudio/SentenceStudio.csproj -f net10.0-maccatalyst
+dotnet build SentenceStudio/SentenceStudio.csproj -f net10.0-android
+dotnet build SentenceStudio/SentenceStudio.csproj -f net10.0-ios
+dotnet build SentenceStudio/SentenceStudio.csproj -f net10.0-windows10.0.19041.0
+```
+
+### Running the Application
```

Comment:
`dotnet run` only works for android. To be consistent use `dotnet build -t:Run` followed by the TFM and path to the csproj